### ISSUE TITLE
Error when opening new project in existing directory

### ISSUE THREAD

**Issue by thomas3300x** [spyder-ide/spyder#15455]
### What steps will reproduce the problem?

open new project in existing directory, same error if old project has been closed or not.

### Traceback
```python-traceback
--- Logging error ---
Traceback (most recent call last):
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'c7af6354a86b11eb9aad14f6d8a0d885'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\programdata\python39\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "c:\programdata\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\programdata\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Python39\Scripts\spyder.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\programdata\python39\lib\site-packages\spyder\app\start.py", line 213, in main
    mainwindow.main(options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2311, in main
    mainwindow = create_window(app, splash, options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2207, in create_window
    app.exec_()
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('c7af6354a86b11eb9aad14f6d8a0d885',)
--- Logging error ---
Traceback (most recent call last):
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'c7af6354a86b11eb9aad14f6d8a0d885'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\programdata\python39\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "c:\programdata\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\programdata\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Python39\Scripts\spyder.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\programdata\python39\lib\site-packages\spyder\app\start.py", line 213, in main
    mainwindow.main(options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2311, in main
    mainwindow = create_window(app, splash, options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2207, in create_window
    app.exec_()
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('c7af6354a86b11eb9aad14f6d8a0d885',)
--- Logging error ---
Traceback (most recent call last):
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'c7af6354a86b11eb9aad14f6d8a0d885'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\programdata\python39\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "c:\programdata\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\programdata\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Python39\Scripts\spyder.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\programdata\python39\lib\site-packages\spyder\app\start.py", line 213, in main
    mainwindow.main(options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2311, in main
    mainwindow = create_window(app, splash, options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2207, in create_window
    app.exec_()
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('c7af6354a86b11eb9aad14f6d8a0d885',)
--- Logging error ---
Traceback (most recent call last):
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'c7af6354a86b11eb9aad14f6d8a0d885'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\programdata\python39\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "c:\programdata\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\programdata\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Python39\Scripts\spyder.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\programdata\python39\lib\site-packages\spyder\app\start.py", line 213, in main
    mainwindow.main(options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2311, in main
    mainwindow = create_window(app, splash, options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2207, in create_window
    app.exec_()
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('c7af6354a86b11eb9aad14f6d8a0d885',)
--- Logging error ---
Traceback (most recent call last):
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'c7af6354a86b11eb9aad14f6d8a0d885'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\programdata\python39\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "c:\programdata\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\programdata\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Python39\Scripts\spyder.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\programdata\python39\lib\site-packages\spyder\app\start.py", line 213, in main
    mainwindow.main(options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2311, in main
    mainwindow = create_window(app, splash, options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2207, in create_window
    app.exec_()
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('c7af6354a86b11eb9aad14f6d8a0d885',)
--- Logging error ---
Traceback (most recent call last):
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'c7af6354a86b11eb9aad14f6d8a0d885'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\programdata\python39\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "c:\programdata\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\programdata\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Python39\Scripts\spyder.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\programdata\python39\lib\site-packages\spyder\app\start.py", line 213, in main
    mainwindow.main(options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2311, in main
    mainwindow = create_window(app, splash, options, args)
  File "c:\programdata\python39\lib\site-packages\spyder\app\mainwindow.py", line 2207, in create_window
    app.exec_()
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "c:\programdata\python39\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('c7af6354a86b11eb9aad14f6d8a0d885',)
```

## Versions

* Spyder version: 5.0.1 
* Python version: 3.9.4
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
setuptools >=39.0.0           :  56.0.0 (OK)
sphinx >=0.6.6                :  3.5.4 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  None (NOK)
numpy >=1.7                   :  None (NOK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by TBSenseo** [comment]
I encountered the same issue when opening a different project. I have the same software versions as above.
In the issue report window, I also had problems to type the "steps to reproduce". I there deleted the existing text and then started to type my text. From this text only the beginning letter was accepted. After this the description box did not accept more characters.

My pip lists the following:

pip list
Package                       Version
----------------------------- ---------
alabaster                     0.7.12
anyio                         2.2.0
appdirs                       1.4.4
argon2-cffi                   20.1.0
arrow                         1.1.0
astroid                       2.5.6
async-generator               1.10
atomicwrites                  1.4.0
attrs                         20.3.0
autopep8                      1.5.5
Babel                         2.9.1
backcall                      0.2.0
bcrypt                        3.2.0
binaryornot                   0.4.4
black                         21.4b2
bleach                        3.3.0
certifi                       2020.12.5
cffi                          1.14.5
chardet                       4.0.0
click                         7.1.2
cloudpickle                   1.6.0
colorama                      0.4.4
cookiecutter                  1.7.2
cryptography                  3.4.7
cycler                        0.10.0
databases                     0.4.3
decorator                     4.4.2
defusedxml                    0.7.1
deprecation                   2.1.0
diff-match-patch              20200713
docutils                      0.16
entrypoints                   0.3
fast-histogram                0.9
flake8                        3.8.4
idna                          2.10
imagesize                     1.2.0
importlib-metadata            4.0.1
inflection                    0.5.1
intervaltree                  3.1.0
ipykernel                     5.5.3
ipython                       7.22.0
ipython-genutils              0.2.0
isort                         5.8.0
jedi                          0.17.2
Jinja2                        2.11.3
jinja2-time                   0.2.0
json5                         0.9.5
jsonschema                    3.2.0
jupyter-client                6.1.12
jupyter-core                  4.7.1
jupyter-packaging             0.9.2
jupyter-server                1.6.4
jupyterlab                    3.0.14
jupyterlab-pygments           0.1.2
jupyterlab-server             2.5.0
keyring                       23.0.1
kiwisolver                    1.3.1
lazy-object-proxy             1.6.0
MarkupSafe                    1.1.1
matplotlib                    3.4.1
mccabe                        0.6.1
mistune                       0.8.4
mpl-scatter-density           0.7
mpld3                         0.5.2
mpmath                        1.2.1
multidict                     5.1.0
mypy-extensions               0.4.3
nbclassic                     0.2.7
nbclient                      0.5.3
nbconvert                     6.0.7
nbformat                      5.1.3
nest-asyncio                  1.5.1
nlopt                         2.7.0
notebook                      6.3.0
numexpr                       2.7.3
numpy                         1.20.2
numpydoc                      1.1.0
packaging                     20.9
pandocfilters                 1.4.3
paramiko                      2.7.2
parso                         0.7.0
pathspec                      0.8.1
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        8.2.0
pip                           21.1
plotly                        4.14.3
pluggy                        0.13.1
poyo                          0.5.0
prometheus-client             0.10.1
prompt-toolkit                3.0.18
psutil                        5.8.0
ptyprocess                    0.7.0
pycodestyle                   2.6.0
pycparser                     2.20
pydocstyle                    6.0.0
pyflakes                      2.2.0
Pygments                      2.8.1
pylint                        2.8.2
pyls-black                    0.4.6
pyls-spyder                   0.4.0
PyNaCl                        1.4.0
pyparsing                     2.4.7
PyQt5                         5.12.3
PyQt5-sip                     12.8.1
PyQtWebEngine                 5.12.1
pyrsistent                    0.17.3
python-dateutil               2.8.1
python-jsonrpc-server         0.4.0
python-language-server        0.36.2
python-lsp-jsonrpc            1.0.0
python-lsp-server             1.0.1
python-slugify                4.0.1
pytz                          2021.1
pywin32                       300
pywin32-ctypes                0.2.0
pywinpty                      1.0.1
pyzmq                         22.0.3
QDarkStyle                    3.0.2
qstylizer                     0.2.0
QtAwesome                     1.0.2
qtconsole                     5.0.3
QtPy                          1.9.0
regex                         2021.4.4
requests                      2.25.1
retrying                      1.3.3
rope                          0.19.0
scipy                         1.6.3
Send2Trash                    1.5.0
setuptools                    56.0.0
six                           1.15.0
sniffio                       1.2.0
snowballstemmer               2.1.0
sortedcontainers              2.3.0
Sphinx                        3.5.4
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        1.0.3
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.4
spyder                        5.0.1
spyder-kernels                2.0.1
SQLAlchemy                    1.3.24
sympy                         1.8
terminado                     0.9.4
testpath                      0.4.4
text-unidecode                1.3
textdistance                  4.2.1
three-merge                   0.1.1
tinycss2                      1.1.0
toml                          0.10.2
tomlkit                       0.7.0
tornado                       6.1
traitlets                     5.0.5
ujson                         4.0.2
urllib3                       1.26.4
watchdog                      1.0.2
wcwidth                       0.2.5
webencodings                  0.5.1
wheel                         0.36.2
wrapt                         1.12.1
wxPython                      4.1.1
XlsxWriter                    1.4.0
yapf                          0.31.0
zipp                          3.4.1

**Comment by steff456** [comment]
Hi @thomas3300x,

It seems this is a duplicate of #15356, that is already fixed by #15462 and will be part of our next version that is going to be released soon. Please keep up posted in our repo and our social media to update to it once it's available.

Thanks for reporting!
